install_files/ansible-base/roles/common/tasks/create_users.yml

```diff
@@ -33,19 +33,3 @@
   tags:
     - users
     - environment
-
-# Backwards-compatibility. Previously, the SecureDrop bashrc additions
```

Comment:
Questioning the comment here: we know from the 0.11.0 upgrades that we have instances that were installed prior to 0.4 that have not run the playbooks since. In which case, we do want to keep this task in for the next time they run `securedrop-admin install`. This PR should remove the part of the comment saying that we can remove this task post 0.4, as it's assuming that admins run the playbooks each release. 

tl;dr modifications to system config implemented only in Ansible will need to stay in the playbooks basically forever due to the long tail of instances not running playbooks in possibly years